src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -798,10 +802,10 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
         const id = tooltip.id;
         const row = tooltip.row - virtualized.offset.rows;
 
-        const { r1c1, tooltip: t } = this.refs as { [key: string]: any };
+        const { table, tooltip: t } = this.refs as { [key: string]: any };
```

Comment:
We want to query the whole table, not just the lower right fragment (which is the same as checking the whole table when nothing is fixed)